packages/superset-ui-translation/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "@superset-ui/translation",
+  "version": "0.0.0",
+  "description": "Superset UI translation",
+  "sideEffects": false,
+  "main": "lib/index.js",
+  "module": "esm/index.js",
+  "files": [
+    "esm",
+    "lib"
+  ],
+  "scripts": {
+    "build:cjs": "beemo babel ./src --out-dir lib/ --minify",
+    "build:esm": "beemo babel ./src --out-dir esm/ --esm --minify",
+    "build": "yarn run build:cjs && yarn run build:esm",
+    "dev": "beemo babel --watch ./src --out-dir esm/ --esm",
+    "jest": "beemo jest --color --coverage",
+    "eslint": "beemo eslint \"./{src,test}/**/*.{js,jsx,json,md}\"",
+    "lint": "yarn run prettier && yarn run eslint",
+    "lint:fix": "yarn run prettier --write && yarn run eslint --fix",
+    "test": "yarn run jest",
+    "prettier": "beemo prettier \"./{src,test}/**/*.{js,jsx,json,md}\"",
+    "prepublish": "yarn run build"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/apache-superset/superset-ui.git"
+  },
+  "keywords": [
+    "superset",
+    "client",
+    "translation"
+  ],
+  "author": "",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/apache-superset/superset-ui/issues"
+  },
+  "homepage": "https://github.com/apache-superset/superset-ui#readme",
+  "devDependencies": {
+    "@data-ui/build-config": "^0.0.23",
+    "react": "^16.4.1"
+  },
+  "dependencies": {
+    "@babel/runtime": "^7.1.2",
+    "jed": "^1.1.1",
+    "sprintf-js": "^1.1.1"
+  },
+  "peerDependencies": {
+    "react": "^16.4.1"
+  },
+  "beemo": {
+    "module": "@data-ui/build-config",
+    "drivers": [
+      "babel",
+      "eslint",
+      {
+        "driver": "jest",
+        "env": {
+          "NODE_ENV": "test"
+        }
+      },
+      "prettier"
+    ],
+    "eslint": {
```

Comment:
It's not a big deal since my PR is moving this config to the root but you can prob remove the app-specific `beemo` config here for this since this was for the promises in `@superset-ui/core`

Reply:
Cool